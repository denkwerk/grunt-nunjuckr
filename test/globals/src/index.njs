<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8">
    <title>Nunjuckr simple test</title>
</head>
<body>

    <h1>This is a simple test for globals in nunjuckr.</h1>

    <h2>{{ globalTitle }}</h2>

</body>
</html>
