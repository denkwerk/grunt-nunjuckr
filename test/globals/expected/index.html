<!DOCTYPE html>
<html>
<head>
    <meta charset="UTF-8">
    <title>Nunjuckr simple test</title>
</head>
<body>

    <h1>This is a simple test for globals in nunjuckr.</h1>

    <h2>This is a global title</h2>

</body>
</html>
